Zowe Explorer Mission

```diff
@@ -0,0 +1,8 @@
+As the mission statement got lost in the formation of the squad process, I'm taking a stab at defining the mission/charter for consideration by the newly formed squad. 
```

Comment:
I would leave this out as it seems more like a comment on the reason for the change. That can be in the PR description.

Reply:
Done.